pkg/java/pom/parse.go

```diff
@@ -510,6 +510,13 @@ func (p *parser) tryRelativePath(parentArtifact artifact, currentPath, relativeP
 		return nil, err
 	}
 
+	// We can inherit GroupID from parent (requires p.analyze function to get GroupID)
+	// Version can contain a property (requires p.analyze function to get version)
+	// But ArtifactID must be the same for parent and base poms
```

Comment:
What do you mean? The parent artifact id and the base artifact id won't match.
Do you mean, "The artifact id of the parent pom in the base pom and the actual artifact id in the parent pom must match."?

Reply:
Right!

But i see that this comment may be confusing.
I updated comment in https://github.com/aquasecurity/go-dep-parser/pull/267/commits/cedc6ceb39982a8117b1daa533f5f6c43c87e9be :
Take a look, please.